src/opera_disp_tms/__main__.py

```diff
@@ -1,82 +1,28 @@
 """OPERA-DISP Tile Map Service Generator"""
 
 import argparse
-from datetime import datetime
-from pathlib import Path
-
-from opera_disp_tms.create_measurement_geotiff import create_measurement_geotiff
-from opera_disp_tms.create_tile_map import create_tile_map
-from opera_disp_tms.utils import upload_dir_to_s3
-
-
-class Date(argparse.Action):
-    def __call__(self, parser, namespace, values, option_string=None):
-        try:
-            value = datetime.strptime(values, '%Y%m%d')
-        except ValueError:
-            parser.error(f'{self.dest} must be formatted YYYYMMDD, e.g. 20211231')
-        setattr(namespace, self.dest, value)
-
-
-class Frames(argparse.Action):
-    def __call__(self, parser, namespace, values, option_string=None):
-        try:
-            frames = [int(item) for sublist in values for item in sublist]
-        except ValueError:
-            parser.error(f'{self.dest} values must be integers between 1 and 46986')
-        for frame in frames:
-            if not (1 <= frame <= 46986):
-                parser.error(f'{self.dest} value {frame} must be between 1 and 46986')
-        setattr(namespace, self.dest, frames)
```

Comment:
Should we preserve the `Date` and `Frames` argument validation for `create_measurement_geotiff.py`?

Reply:
Or at least min/max for the frame value?